.openshift-ci/dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+# These versions match those in stackrox-test-0.3.62
```

Comment:
How do we keep this in sync? :thinking:

Reply:
I can at least make the comment better here and in the test image repo.

Reply:
https://github.com/stackrox/rox-ci-image/pull/194